### ISSUE TITLE
Dasharo HCL report does not dump ACPI

### ISSUE THREAD

**Issue by miczyg1** [Dasharo/dasharo-issues#1168]
### Component

Dasharo Tools Suite

### Device

Protectli VP6670

### Dasharo version

Any

### Dasharo Tools Suite version

v2.1.0

### Test case ID

HCL report

### Brief summary

The acpidump.log and its respective err file is empty

### How reproducible

Always

### How to reproduce

Perform HCl report on VP6670 on any firmware

### Expected behavior

The acpidump.log contains the ACPI tables.

### Actual behavior

The acpidump.log and its respective err file is empty

### Screenshots

Example logs: https://cloud.3mdeb.com/index.php/f/794023

**Comment by m-iwanicki** [comment]
Fixed by https://github.com/Dasharo/dts-scripts/pull/56. Tested fix on `Protectli VP6670 v0.9.0-rc5`:

```
bash-5.2# ls logs/acpi
apic.dat  dmar.dat  dsdt.dat  facp.dat  facs.dat  hpet.dat  lpit.dat  mcfg.dat  spcr.dat  ssdt.dat  tpm2.dat  wdat.dat
apic.dsl  dmar.dsl  dsdt.dsl  facp.dsl  facs.dsl  hpet.dsl  lpit.dsl  mcfg.dsl  spcr.dsl  ssdt.dsl  tpm2.dsl  wdat.dsl
bash-5.2# head logs/acpidump.log
SSDT @ 0x0000000000000000
    0000: 53 53 44 54 EB 37 00 00 02 7E 43 4F 52 45 76 34  SSDT.7...~COREv4
    0010: 43 4F 52 45 42 4F 4F 54 00 00 00 00 43 4F 52 45  COREBOOT....CORE
    0020: 28 06 23 20 5B 82 48 04 43 54 42 4C 08 5F 48 49  (.# [.H.CTBL._HI
    0030: 44 0D 42 4F 4F 54 30 30 30 30 00 08 5F 55 49 44  D.BOOT0000.._UID
    0040: 00 08 5F 53 54 41 0A 0B 08 5F 43 52 53 11 20 0B  .._STA..._CRS. .
    0050: 1C 00 87 17 00 00 1C 02 00 00 00 00 00 10 92 76  ...............v
    0060: FF 8F 92 76 00 00 00 00 00 80 00 00 79 00 10 48  ...v........y..H
    0070: 68 5C 2E 5F 53 42 5F 50 43 49 30 14 4B 67 5F 50  h\._SB_PCI0.Kg_P
    0080: 52 54 00 A0 46 3F 50 49 43 4D A4 12 4E 3E 3B 12  RT..F?PICM..N>;.
```

**Comment by miczyg1** [comment]
Approved the PR, since I saw that `$ACPIDUMP` was probably not defined anywhere when the issue occurred.
